doc/nix-local-build.rst

```diff
@@ -146,11 +146,11 @@ must be built per-project, versus external packages, which can be cached
 across projects. To be more precise:
 
 1. A **local package** is one that is listed explicitly in the
-   ``packages``, ``optional-packages`` or ``extra-packages`` field of a
-   project. Usually, these refer to packages whose source code lives
-   directly in a folder in your project. But you can list an
-   arbitrary Hackage package in :cfg-field:`packages`
-   to force it to be treated as local.
+   ``packages``, ``optional-packages`` and ``extra-packages`` fields of a
+   project. They refer to packages, whose source code usually lives
```

Comment:
Not a native speaker but I feel the comma is unnecessary

Reply:
I rewrote the sentence, because it was too complicated anyway.